app/templates/components/preprint-form-header.hbs

```diff
@@ -89,12 +94,6 @@
             {{/if}}
         {{/if}}
 
-        {{#if (eq name 'location_of_preprint')}}
-            {{#if node}}
-                <p class='text-muted'> <span class='lock-span p-r-xs'><i class='fa fa-lock'></i></span> {{get-ancestor-descriptor node}} {{node.title}} </p>
-            {{/if}}
-        {{/if}}
-
         {{#if (eq name 'choose_project')}}
```

Comment:
Can be deferred or ignored:

This could use helper text if no files exist
<img width="931" alt="screen shot 2018-10-02 at 10 18 07 pm" src="https://user-images.githubusercontent.com/1322421/46386734-21893d80-c691-11e8-8e8e-5c0de5e3c968.png">

Reply:
If no files found:
<img width="916" alt="screen shot 2018-10-03 at 9 28 29 pm" src="https://user-images.githubusercontent.com/9755598/46449803-8c02b200-c753-11e8-9192-57fa0b66e722.png">